### DISCUSSION TITLE
how to check char is inhouse or not？```py

### DISCUSSION THREAD

**Discussion by varbakett** [Reetus/ClassicAssist#293] [Ideas]
can check inhouse？

**Comment by d0tsd0tsd0ts** [comment]
Hey @varbakett, how are u ?? Fine ?  I guess that u can get script answer faster by discord. This channel is used to report bug or ask feature...

**Comment by varbakett** [comment]
@d0tsd0tsd0ts   can u give me discord  link.

**Comment by d0tsd0tsd0ts** [comment]
https://discord.com/channels/458277173208547350/685864489266053193

**Comment by varbakett** [comment]
channels can't find... 
can u tell me search words.
add me "vatfff"

**Comment by varbakett** [comment]
vatfff#5279
